dropwizard-validation/src/main/java/io/dropwizard/validation/MaxDuration.java

```diff
@@ -41,4 +41,9 @@
      * @return unit of the value the element must be higher or equal to
      */
     TimeUnit unit() default TimeUnit.SECONDS;
+
+    /**
+     * @return if the validation should include the set value (default is true).
```

Comment:
Perhaps something worded like the following, for this and the other equivalent javadocs.

"True if the validation is to allow values equal to {@code value}. False if the validation is to be exclusive. Defaults to {@code true}."